recipe/meta.yaml

```diff
@@ -47,14 +43,12 @@ outputs:
         - python
         - cycler >=0.10
         - fonttools >=4.22.0
-        - freetype >=2.3
```

Comment:
Why was this run dependency removed? Seems to me that it will stil be used.

Reply:
I wasn't sure about this - the `system_freetype = true` line `build_base.sh` means that the system is expected to have `freetype` installed; no point in importing the library in the `run` section then, right? Note that if this were wrong the package fails to build.

Reply:
I think this should still be in the run section as freetype is a package and not a cdt.

Reply:
Fixed